app/controllers/entries_controller.rb

```diff
@@ -31,7 +33,7 @@ def new
 
   def create
     @entry = Entry.new(entry_params)
-
+    @list = @entry.list
```

Comment:
RIght now, the list doesn't exist.  You're creating an Entry but you're relying on list id to be passed into the entry_params.

What should happen if we try to create an entry where the list doesn't exist?  Consequently, an entry in which you are editing to not have a list at all would also be something we should test against.